3drepo/3drepo.io
# Issue 4493 - improve performance in rendering many containers

This fixes #4493 

#### Description
The problem was not just with containers, but also with federations displaying and in the edit/create federation (second step, when it comes to select the containers to add).

It seems that the root cause was actually the skeleton component, which has been mostly replace by the MUI skeleton component which performs better.

The loading time significantly improved and the involved pages now are much faster at loading. 
However, we may want to consider the use of pagination to improve this even further

#### Test cases
upload 100 containers (I used 200 to stress test it) then:
- load the containers tab;
- load the federations tab;
- load the edit federation modal section;

The loading time should be acceptable
